man/docker-lvm-plugin.8.md

```diff
@@ -1,68 +1,83 @@
-% DOCKER-LVM-PLUGIN(8) 
-% Shishir Mahajan 
+% DOCKER-LVM-PLUGIN(8)
+% Shishir Mahajan
 % FEBRUARY 2016
 # NAME
 docker-lvm-plugin - Docker Volume Driver for lvm volumes
 
 # SYNOPSIS
-**docker-lvm-plugin**
-[**-debug**]
-[**-version**]
+**docker volume create -d lvm**
+[**--opt size**]
+[**--opt thinpool**]
+[**--opt snapshot**]
+[**--opt keyfile**]
+[**--opt vg**]
 
```

Comment:
`SYNOPSIS` should not be changed. That lists the options to docker-lvm-plugin daemon {debug, version}
We don't need to list docker volume create options here.